src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/Example1.txt

```diff
@@ -23,4 +23,6 @@ List<T> list = ImmutableList.Builder<T>::new;
 sort(list, Comparable::<String>compareTo);
 // Constructor call
 MyClass obj = new <String>MyClass();
+// Record header
+record License<T>() {}
```

Comment:
Have kept the examples in the non-enabled way as GenericWhitespaceExample is being updated in the PR [#1464](https://github.com/checkstyle/checkstyle/pull/14364), if that happens to get merged before this one does, I will enable the examples else it can be updated in that PR.